less/mixins.less

```diff
@@ -1 +1,14 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.createButton(@bor, @wide, @font, @padl, @padr, @padt, @padb, @br, @bg, @col, @txt,){
+    border: @bor;
+    width: @wide;
+    font-size: @font;
+    padding-left: @padl;
+    padding-right: @padr;
+    padding-top: @padt;
+    padding-bottom: @padb;
+    border-radius: @br;
+    background-color: @bg;
+    color: @col;
+    text-align: @txt;
+}
```

Comment:
This is a complex mixin. It's nice to know that you can do it but it doesn't look practical.